### PR TITLE
Operator needs to provide an empty sink-filters ConfigMap.

### LINKED ISSUE #300: Operator needs to provide an empty sink-filters ConfigMap
The operator needs to include an empty sink-filters `ConfigMap`.  It will create one during reconciliation if it it doesn't exist, but that won't happen until a `KubeArchiveConfig` resource is created and reconciled.  The sink needs one to mount.

### PR DESCRIPTION
**Pull request by ggallen**
## Which Issue(s) this Pull Request Fixes

Resolves #300

## Release Note

```release-note
NONE
```

### CODE REVIEW

File: charts/kubearchive/templates/operator/sink_filters.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
```
Comment (beerparty) [inline charts/kubearchive/templates/operator/sink_filters.yaml:1]: You are missing the YAML header.

```suggestion
---
apiVersion: v1
```

File: charts/kubearchive/templates/operator/sink_filters.yaml
Comment (beerparty) [inline charts/kubearchive/templates/operator/sink_filters.yaml:1]: Should this file (and the other templates) have the copyright and license notice?

File: charts/kubearchive/templates/operator/sink_filters.yaml (hunk 1 above)
Comment (ggallen) [reply charts/kubearchive/templates/operator/sink_filters.yaml:2]: Fixed.

File: charts/kubearchive/templates/operator/sink_filters.yaml
Comment (ggallen) [reply charts/kubearchive/templates/operator/sink_filters.yaml:1]: That I don't know.  But I don't think any have them, and I think if we want to add them, then that is a different issue.

**Review by skoved — APPROVED** [review]

File: charts/kubearchive/templates/operator/sink_filters.yaml
Comment (beerparty) [reply charts/kubearchive/templates/operator/sink_filters.yaml:1]: We should discuss with the group. I will resolve this since it is a bigger issue.

**Review by beerparty — APPROVED** [review]

**Pull request merged**